### ISSUE TITLE
peewee.OperationalError: near "WITHOUT": syntax error

### ISSUE THREAD

**Issue by shjoshi95** [ranaroussi/yfinance#1825]
### Describe bug

When trying to access the .info property of the Ticker object, I am getting the following error.

### Simple code that reproduces your problem

yf_ticker = yf.Ticker(ticker)
info = yf_ticker.info

### Debug log

 info = yf_ticker.info
           ^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/ticker.py", line 154, in info
    return self.get_info()
           ^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/base.py", line 1779, in get_info
    data = self._quote.info
           ^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/scrapers/quote.py", line 574, in info
    self._fetch_info(self.proxy)
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/scrapers/quote.py", line 649, in _fetch_info
    result = self._fetch(proxy, modules=modules)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/scrapers/quote.py", line 638, in _fetch
    result = self._data.get_raw_json(_QUOTE_SUMMARY_URL_ + f"/{self._symbol}", user_agent_headers=self._data.user_agent_headers, params=params_dict, proxy=proxy)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/data.py", line 399, in get_raw_json
    response = self.get(url, user_agent_headers=user_agent_headers, params=params, proxy=proxy, timeout=timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/data.py", line 348, in get
    cookie, crumb, strategy = self._get_cookie_and_crumb()
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/data.py", line 324, in _get_cookie_and_crumb
    cookie, crumb = self._get_cookie_and_crumb_basic(proxy, timeout)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/data.py", line 213, in _get_cookie_and_crumb_basic
    cookie = self._get_cookie_basic(proxy, timeout)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/data.py", line 156, in _get_cookie_basic
    self._cookie = self._load_cookie_basic()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/data.py", line 142, in _load_cookie_basic
    cookie_dict = cache.get_cookie_cache().lookup('basic')
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/cache.py", line 355, in lookup
    self.initialise()
  File "/home/ec2-user/.local/lib/python3.11/site-packages/yfinance/cache.py", line 347, in initialise
    db.create_tables([_CookieSchema])
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 3441, in create_tables
    model.create_table(**options)
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 6948, in create_table
    cls._schema.create_all(safe, **options)
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 6050, in create_all
    self.create_table(safe, **table_options)
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 5901, in create_table
    self.database.execute(self._create_table(safe=safe, **options))
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 3260, in execute
    return self.execute_sql(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 3250, in execute_sql
    with __exception_wrapper__:
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 3020, in __exit__
    reraise(new_type, new_type(exc_value, *exc_args), traceback)
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 192, in reraise
    raise value.with_traceback(tb)
  File "/home/ec2-user/.local/lib/python3.11/site-packages/peewee.py", line 3252, in execute_sql
    cursor.execute(sql, params or ())
peewee.OperationalError: near "WITHOUT": syntax error

### `yfinance` version

0.2.35

### Python version

3.11

### Operating system

linux (amazon ec2)

**Comment by shjoshi95** [comment]
I tried troubleshooting from a similar looking issue here: https://github.com/ranaroussi/yfinance/issues/1372

But there is no matching distribution of sqlite3 on python3.11 that pip could resolve to.

**Comment by ValueRaider** [comment]
What is version of `sqlite`? At minimum there should be a version bundled in Python.

**Comment by shjoshi95** [comment]
Interestingly, I could not find a version of sqlite or sqlite3. On my local too where this issue does not happen, neither of these packages are installed either.

$ pip show sqlite
WARNING: Package(s) not found: sqlite
$ pip show sqlite3
WARNING: Package(s) not found: sqlite3

**Comment by ValueRaider** [comment]
Bundled in Python means ignore PIP. Just do `import sqlite ; sqlite.__version__`

**Comment by shjoshi95** [comment]
I see, thank you for that clarification! In my python3.11, I only was able to import sqlite3, and the version is 2.6.0 `import sqlite3 ; sqlite3.version`

**Comment by ValueRaider** [comment]
Try git branch `fix/peewee-with-old-sqlite` #1080. Not tested it with your old sqlite, hoping to avoid reproducing your setup. If still fails, then look at the code change, you might be able to get it working.

**Comment by shjoshi95** [comment]
Actually, I ended up discovering the problem and solution. Firstly, on sqlite3, `sqlite3.version` doesn't mean anything, it will be the same everywhere. Instead, `sqlite3.sqlite_version` gives the true version. When I checked this on my VM, the version was only 3.7.17, which goes back to 2013, while my local machine was mostly up to date at 3.39.4, which is from 2022. Thus, I upgraded the sqlite3 binary on my VM to the latest and magically everything worked. 

Thanks again @ValueRaider for your support, without your suggestion to start looking into the versions, I would not have discovered this fix. As always, am appreciative of all the work that goes into keeping the yfinance project alive.
